src/targets/gpu/target.cpp

```diff
@@ -92,7 +92,6 @@ std::vector<pass> target::get_passes(migraphx::context& gctx, const compile_opti
     unsupported_types.erase(shape::type_t::half_type);
     unsupported_types.erase(shape::type_t::bool_type);
     unsupported_types.erase(shape::type_t::int8_type);
-    unsupported_types.erase(shape::type_t::uint8_type);
```

Comment:
Revert this change. 

`uint8_type` is supported type. 

If you erase that from unsupported_types then eliminate_data_type would try to convert quantizelinear with uint8 to float. 

I think We should only convert uint8 type quant_dot and quant_convolution to supported type.

Reply:
Reverted